src/algorithms/derivatives/hamiltonian_derivatives.jl

```diff
@@ -138,9 +138,12 @@ function AC_hamiltonian(
         finished = removeunit(GL[end], 2)
     end
 
-    # continuing
-    A = W.A
-    continuing = (GL[2:(end - 1)], A, GR[2:(end - 1)])
+    if nonzero_length(W.A) > 0
+        @tensor GLW[-1 -2 -3; -4 -5] ≔ GL[2:(end - 1)][-1 1; -4] * W.A[1 -2; -5 -3]
```

Comment:
```suggestion
        @plansor GLW[-1 -2 -3; -4 -5] ≔ GL[2:(end - 1)][-1 1; -4] * W.A[1 -2; -5 -3]
```
This is a fermion/anyon thing, I can explain tomorrow if you like.